pgcli/key_bindings.py

```diff
@@ -42,7 +42,11 @@ def _(event):
         Force autocompletion at cursor.
         """
         _logger.debug('Detected <Tab> key.')
-        event.cli.current_buffer.complete_next()
+        b = event.cli.current_buffer
+        if b.complete_state:
```

Comment:
Wow github totally clobbered my comment. I meant to ask if it's possible to overload the `Enter` key to select the first suggestion. If you re read my comment by prefixing the word 'key' with `Enter` key, it'll make sense. 

I used `<Enter>` and I think github helpfully cleaned anything that looked like a html tag.